### PR TITLE
Topics crud

### PR DESCRIPTION
**Pull request by advantagevarnson**
@droberts84 hey so i think all the crud action works well but im having trouble with layout?

any tips on how to create boxed topics/bookmarks similar to the wireframe?

### CODE REVIEW

**Comment by droberts84** [comment]
Checkout getbootstrap.com for ideas. It doesn't need to look the same as
the wire frame.. Just make it look nice

File: app/controllers/topics_controller.rb [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
   end
 
   def new
+    @topic = Topic.new
   end
 
   def show
-    @bookmarks = @topic.bookmarks.paginate(page: params[:page], per_page: 4)
+    @topic = Topic.find(params[:id])
+    @bookmarks = @topic.bookmarks
```
Comment (droberts84) [inline app/controllers/topics_controller.rb]: no need for bookmarks yet

File: app/models/topic.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Topic < ActiveRecord::Base
+  belongs_to :user
+    has_many :bookmarks
```
Comment (droberts84) [inline app/models/topic.rb]: fix the indentation

File: config/routes.rb [hunk 3]
```diff
@@ -3,6 +3,7 @@
   get 'topics/index'
 
   devise_for :users
+    resources :topics
```
Comment (droberts84) [inline config/routes.rb]: this should not be indented

File: spec/models/bookmark_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Bookmark, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (droberts84) [inline spec/models/bookmark_spec.rb]: Make sure you either add some tests or delete

File: spec/models/topic_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (droberts84) [inline spec/models/topic_spec.rb]: Make sure you either add some tests or delete

File: app/controllers/topics_controller.rb (hunk 1 above)
Comment (advantagevarnson) [inline app/controllers/topics_controller.rb]: in the spec it says: 

```
Implement the show action in topics_controller.rb and display the bookmarks in app/views/topics/show.html.erb
```

Comment (droberts84) [inline app/controllers/topics_controller.rb]: ah, ok.

**Pull request merged**